addon/components/frost-fixed-table.js

```diff
@@ -82,6 +85,17 @@ export default Component.extend({
   @readOnly
   @computed('columns')
   /**
+   * Pre-computed indices
+   * @param {Column[]} columns - the column data we want to present
+   * @returns {Column[]} indexed - columns with an 'index' property added
+   */
+  indexedColumns (columns) {
+    return columns.map((column, index) => Object.assign({index}, column))
```

Comment:
Currently since `columns` isn't a required property this could technically choke on `columns.map` if/when columns isn't present. Of course a table doesn't make much sense without columns hence my other comment to make it required.